src/main/java/redis/clients/jedis/JedisPooled.java

```diff
@@ -394,6 +395,11 @@ public JedisPooled(PooledConnectionProvider provider) {
     super(provider);
   }
 
+  public JedisPooled(CommandExecutor executor, PooledConnectionProvider provider, CommandObjects commandObjects,
+                     RedisProtocol redisProtocol) {
```

Comment:
1. We do not plan to change CommandExecutor from here.
```suggestion
  public JedisPooled(PooledConnectionProvider provider, CommandObjects commandObjects, RedisProtocol redisProtocol) {
```
2. Add a JavaDoc stating redisProtocol will be set into commandObjects.